packages/@glimmer/compiler/lib/template-compiler.ts

```diff
@@ -18,8 +18,6 @@ function isTrustedValue(value: any) {
   return value.escaped !== undefined && !value.escaped;
 }
 
-export const THIS = 0;
```

Comment:
Nope, it was dead code. Unless it’s some kind of public API but then since the entire codebase doesn’t do anything with it I don’t know what that would do either.